packages/realm-server/main.ts

```diff
@@ -337,19 +333,3 @@ async function startWorker() {
     process.exit(-2);
   }
 }
-
-// In case there are no permissions in the database, seed the realm with default permissions:
-// Base realm: read permissions for everyone
-// Other realms: read permissions for everyone, read/write permissions for signed up users
-async function seedRealmPermissions(dbAdapter: PgAdapter, realmURL: URL) {
-  if (!(await permissionsExist(dbAdapter, realmURL))) {
-    if (realmURL.href === 'https://cardstack.com/base/') {
-      await insertPermissions(dbAdapter, realmURL, { '*': ['read'] });
-    } else {
-      await insertPermissions(dbAdapter, realmURL, {
-        '*': ['read'],
-        users: ['read', 'write'],
-      });
-    }
-  }
-}
```

Comment:
This PR adds permissions to all the public realms that we have. Dynamically created realms already get permissions assigned. Creating public realms is very much a one off activity (a not insignificant PR needs to be filed). As part of this the permissions necessary for each public realm should be included in the PR (I took care of the forthcoming catalog realm in this PR). if there is a new way that public realms are created in the future we can revisit this. however, I believe that permissions should always be assigned when the realm is created, not after the fact like this.